Common/Source/lk8000.cpp

```diff
@@ -571,7 +574,7 @@ bool Startup(const TCHAR* szCmdLine) {
   GlobalRunning = true;
 	
 	InitAirspaceSonar();
-
+	StartIGCReadThread() ;
```

Comment:
start new thread even if you don't use flarm device is bad idea...

Reply:
OK moved to:
BOOL CDevFlarm::Open( PDeviceDescriptor_t d)